src/session.ts

```diff
@@ -1,5 +1,5 @@
 /*!
- * Copyright 2018 Google Inc. All Rights Reserved.
+ * Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
dont introduce this change, if you are introducing this change then fix it properly check in other new header file and make change accordingly.

Reply:
Okay

Reply:
Please let me clear should I keep same as old copyright year or update to current year.

Reply:
You don't need to update Copyrights, Leave it as it is.  If this is a **new file** then you have to add Copyright with this:
```
/*
 * Copyright 2018 Google LLC. All Rights Reserved.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
```

Notice this is Inc is LLC now.

Reply:
Okay. Got it. Thank you Rahul and Ajay.